desktop_server/src/com/mygdx/game/server/di/GeneratorBindingsModule.java

```diff
@@ -0,0 +1,24 @@
+package com.mygdx.game.server.di;
+
+import com.mygdx.game.server.initialize.generators.BasicSubfieldMapGenerator;
+import com.mygdx.game.server.initialize.generators.BotWinningFieldMapGenerator;
+import com.mygdx.game.server.initialize.generators.MapGenerator;
+import com.mygdx.game.server.initialize.generators.SubfieldMapGenerator;
+import dagger.Binds;
+import dagger.Module;
+import dagger.multibindings.IntoSet;
+
+@Module
```

Comment:
Nie za bardzo jeszcze to rozumiem, ale tak średnio mi sięto podoba

Reply:
jest równoważne z robieniem listy w providerze i tworzeniu z tego Seta i przekazywaniem tego do konstruktora